sites/postcss-preset-env/netlify.toml

```diff
@@ -1,2 +1,7 @@
 [build]
 	ignore = "exit 0"
+
+[[redirects]]
+	from = "/feed/"
+	to = "/feed/feed.xml"
+	status = 200
```

Comment:
_purely out of curiosity_

Why does a redirect have a status of 200?
I have no idea how the netlify feature works :)